lib/monaco-adapter.js

```diff
@@ -136,48 +163,6 @@ firepad.MonacoAdapter = (function () {
     return this.grabDocumentState();
```

Comment:
This is not required to be returned for Monaco. Just update the state and leave it.

Reply:
Thanks, done.